apollo-ios/Sources/Apollo/RequestChain.swift

```diff
@@ -158,16 +145,7 @@ public struct RequestChain<Request: GraphQLRequest>: Sendable {
 
             // Cache miss
 
-          } catch {
-            #warning(
```

Comment:
After considering this further, I don't think we should make that change. If the user's cache policy asks for network data on a cache failure, it shouldn't matter if the cache failed do to invalid data or just a cache miss, either way, the correct way to handle that would be to re-fetch the data from the network and re-hydrate the cache with the correct data.